include/curl/curl.h

```diff
@@ -1352,9 +1351,7 @@ typedef enum {
   /* Max amount of cached alive connections */
   CURLOPT(CURLOPT_MAXCONNECTS, CURLOPTTYPE_LONG, 71),
 
-  /* OBSOLETE, do not use! */
-  CURLOPT(CURLOPT_OBSOLETE72, CURLOPTTYPE_LONG, 72),
-
+  /* 72 = OBSOLETE */
```

Comment:
Noticed in vpckg CI run on CURL HEAD:

`CURLOPT_OBSOLETE72` is still used in the header for

https://github.com/curl/curl/blob/3fc81be44e970215f6ca18ab331ac2050abda58b/include/curl/curl.h#L723

and `CURLOPT_CLOSEPOLICY` is used downstream in curlpp and curlcpp.